pkg/utils/helper.go

```diff
@@ -52,6 +54,20 @@ func IsAzureStackCloud() bool {
 	return strings.EqualFold(cloud, AzureStackCloudName)
 }
 
+// IsKubernetesInDocker returns true if the application is running on KubernetesInDocker (kind)
+func IsKubernetesInDocker() bool {
+	//TODO refactor the conditional logic this check guards into a new "KindClusterOperations" type behind an interface
+	//test the AKS kubeconfig location, if we find something then this isn't a KIND cluster
+	_, err := RunCommandOnHost("ls", "/var/lib/kubelet/kubeconfig")

```

Comment:
they are libraries in golang to do the equivalent as a ls. We should stop using RunCommand

Reply:
As above, will this work via nsenter?